docs/papers/llm-load-balancing-at-scale-chwbl.md

```diff
@@ -0,0 +1,120 @@
+# LLM Load Balancing at Scale: Consistent Hashing with Bounded Loads
+
+Feb 26, 2025
+
+Posted by [Nick Stogner](https://www.linkedin.com/in/nstogner/), Co-Founder of [Substratus.AI](https://www.substratus.ai/)
+
+## Abstract:
+
+When deploying large language models (LLMs) at scale efficient load balancing is crucial for minimizing latency and maximizing throughput. We explore the application of the Consistent Hashing with Bounded Loads (CHWBL) algorithm in the domain of LLM serving. Our approach dramatically reduces Time To First Token (TTFT) while improving overall Tokens Per Second (TPS) compared to the baseline strategies.
```

Comment:
Maybe we should add some of the results in 1 sentence  here as well. So people see the value of the paper in the abstract.

Reply:
e.g. change this to a number "Our approach dramatically reduces Time To First Token (TTFT)"

Reply:
Agreed, made change